tests/osaka/eip7939_count_leading_zeros/test_count_leading_zeros.py

```diff
@@ -125,15 +125,15 @@ def test_clz_gas_cost(state_test: StateTestFiller, pre: Alloc, fork: Fork):
     tx = Transaction(to=contract_address, sender=sender, gas_limit=200_000)
     post = {
         contract_address: Account(  # Cost measured is CLZ + PUSH1
-            storage={"0x00": fork.gas_costs().G_VERY_LOW}
+            storage={"0x00": fork.gas_costs().G_LOW}
         ),
     }
     state_test(pre=pre, post=post, tx=tx)
 
 
 @pytest.mark.valid_from("Osaka")
 @pytest.mark.parametrize("bits", [0, 64, 128, 255])
-@pytest.mark.parametrize("gas_cost_delta", [-2, -1, 0, 1, 2])
+@pytest.mark.parametrize("gas_cost_delta", list(range(-Spec.CLZ_GAS_COST, Spec.CLZ_GAS_COST + 1)))
```

Comment:
I feel like this range will create too many tests. What if we do something like this, then negate the `gas_cost_delta` field. So `- gas_cost_delta` instead `+ gas_cost_delta`. Wdyt?
```suggestion
@pytest.mark.parametrize("gas_cost_delta", [
    Spec.CLZ_GAS_COST - 1,
    Spec.CLZ_GAS_COST,
    Spec.CLZ_GAS_COST + 1
])
```

Reply:
If we keep using gas_cost_delta, it might make sense to test the values `[-1, 0, 1]`. However, if we directly use:
- Spec.CLZ_GAS_COST - 1
- Spec.CLZ_GAS_COST
- Spec.CLZ_GAS_COST + 1

then the total gas provided, `fork.gas_costs().G_VERY_LOW + Spec.CLZ_GAS_COST + gas_cost_delta`, will always be sufficient.

Considering both the test scope and the original intent, it might be reasonable to keep the original test unchanged.

Reply:
Great point. Sounds good to me! I just don't want to create too many tests regarding the EELS coverage topic during the STEEL call this week.